src/Qwiq.Core.Rest/Extensions.cs

```diff
@@ -8,50 +8,50 @@ namespace Qwiq.Client.Rest
 {
     internal static class Extensions
     {
-        [CanBeNull]
-        [Pure]
-        [ContractAnnotation("null => null; notnull => notnull")]
-        internal static IWorkItem AsProxy([CanBeNull] this WorkItem item)
+
```

Comment:
With the removal of JetBrains.Annotations, there's now a bunch of whitespace. Let's clean this up